internal/kafka/internal/metrics/component_versions_test.go

```diff
@@ -59,3 +59,50 @@ func TestVersionsMetrics_Collect(t *testing.T) {
 		})
 	}
 }
+
+func TestVersionsMetrics_Describe(t *testing.T) {
+	type fields struct {
+		kafkaService services.KafkaService
+	}
+
+	type args struct {
+		ch chan<- *prometheus.Desc
+	}
+
+	tests := []struct {
+		name   string
+		fields fields
+		args   args
+		want   int
+	}{
+		{
+			name: "return",
+			fields: fields{kafkaService: &services.KafkaServiceMock{
+				ListComponentVersionsFunc: func() ([]services.KafkaComponentVersions, error) {
+					return []services.KafkaComponentVersions{
+						{
+							ID:                    "1",
+							ClusterID:             "cluster1",
+							DesiredStrimziVersion: "1.0.1",
+							ActualStrimziVersion:  "1.0.0",
+							StrimziUpgrading:      true,
+						},
+					}, nil
+				},
+			}},
+			args: args{ch: make(chan *prometheus.Desc, 100)},
+			want: 1,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			m := newVersionMetrics(tt.fields.kafkaService)
+			ch := tt.args.ch
+			m.Describe(ch)
+			if len(ch) != tt.want {
+				t.Errorf("expect to have %d metrics but got %d", tt.want, len(ch))
+			}
```

Comment:
let's try to use gomega assertions here. Just like https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/019ed9a849f792938f7a2b442f2ddc0164cb1e98/pkg/auth/context_test.go#L70 for example
```suggestion
			Expect(tt.want).To(Equal(len(ch)))
```
or much better alternative
```suggestion
			Expect(ch).To(HaveLen(tt.want))
```

Reply:
Few other changes to make actually :)